### PR TITLE
fix[rocksdb]: fix error rocksdb default config for CFOptions

### PR DESCRIPTION
**Pull request by AuroraTwinkle**
Descriptions of the changes in this PR:

Fix erros of rocksdb default config
### Motivation
The default rocksdb config has some errors and need fix.

### Changes
1. Fix error rocksdb default config for CFOptions.
2. Update rocksdb config for unit test

### CODE REVIEW

**Review by liangyepianzhou — APPROVED** [review]
LGTM

**Review by shoothzj — APPROVED** [review]

**Review by StevenLuMT — APPROVED** [review]
Good Job, LGTM

**Comment by dlg99** [comment]
The change broke KeyValueStorageRocksDBTest:

```
Error:  org.apache.bookkeeper.bookie.storage.ldb.KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile
Error:    Run 1: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
Error:    Run 2: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
Error:    Run 3: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
```

**Comment by AuroraTwinkle** [comment]
> The change broke KeyValueStorageRocksDBTest:
> 
> ```
> Error:  org.apache.bookkeeper.bookie.storage.ldb.KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile
> Error:    Run 1: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
> Error:    Run 2: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
> Error:    Run 3: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
> ```

Thanks，I will fix all failed Testcases.

**Comment by AuroraTwinkle** [comment]
> The change broke KeyValueStorageRocksDBTest:
> 
> ```
> Error:  org.apache.bookkeeper.bookie.storage.ldb.KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile
> Error:    Run 1: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
> Error:    Run 2: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
> Error:    Run 3: KeyValueStorageRocksDBTest.testRocksDBInitiateWithConfigurationFile:74 expected:<1> but was:<30>
> ```

I have fixed it.

**Review by dlg99 — APPROVED** [review]

File: conf/entry_location_rocksdb.conf.default [hunk 1]
```diff
@@ -26,7 +30,7 @@
  # set by jni: options.setLogFileTimeToRoll
  log_file_time_to_roll=86400
  # set by jni: options.setMaxBackgroundJobs or options.setIncreaseParallelism
- max_background_jobs=2
+ max_background_jobs=32
```
Comment (lhotari) [inline conf/entry_location_rocksdb.conf.default:33]: What's the rationale in making this change in this PR?

Comment (AuroraTwinkle) [reply conf/entry_location_rocksdb.conf.default:33]: > What's the rationale in making this change in this PR?

Hi Lari:
The previous default value for **max_background_jobs** was written incorrectly and was not consistent with the default value in the code, so this PR corrected it. Refer to: https://github.com/apache/bookkeeper/blob/999cd0f2ab14404be4d6c24e388456dbe56bb1a8/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java#L213

Comment (lhotari) [reply conf/entry_location_rocksdb.conf.default:33]: Thanks for explaining @AuroraTwinkle

**Pull request merged**